src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine bulldozer = new Bulldozer();
+
+        Machine truck = new Truck();
+
+        Machine excavator = new Excavator();
+
+        Machine[] arr = {bulldozer, truck, excavator};
```

Comment:
```suggestion
        Machine[] machines = {new Bulldozer(), new Truck(), new Excavator()};
```
arr is always a bad naming :)
new lines between machines are unnecessary